samples/cmake/ozz-animation/README.md

```diff
@@ -0,0 +1,71 @@
+You must clone ozz first:
+
+```
+git clone https://github.com/guillaumeblanc/ozz-animation.git --branch 0.13.0 --depth=1
+mkdir build
+cd build
+${NXDK_DIR}/usr/bin/nxdk-cmake ..
```

Comment:
I found an older comment saying:

> ozz needs CMake to run twice; also the SDL renderer was not tested on Xbox, because it hangs during startup (fails to load skeleton / animation)

Compare https://github.com/JayFoxRox/nxdk/pull/84/files#r611207045